README.md

```diff
@@ -82,6 +82,15 @@ User.create(email: 'xxxxxx', password: 'xxxxxx')
 5. Load test users and sample data: `bundle exec rake db:seed`
 5. Start the application: `bundle exec rails s`
 
+### Local Production Dependency Build
+The steps below will setup your local without dependencies that are not loaded in dev, staging, and production environments.
+1. `bundle install --without development test`
+2. `bundle clean --force`
+3. `bundle exec rails s -p 4000`
```

Comment:
I think what you want is just 1 step - `bundle exec  rails server -e production` is there a reason you want it to run on port 4000?  The only change you'd need to make is to make sure that you're not actually connection to the production database in database.yml

Reply:
port 4000 is just how I run it, removing the option

Reply:
So I tried running with `bundle exec rails server -e production` without the gems `rainbow` or `sass-rails` being installed or listed in Gemfile and the UI portion of GIDS was still functioning normally because they are pulled in by `rubocop`, I had not done `bundle install --without development test; bundle clean --force;` just `bundle install; bundle clean --force;`.

The original issue that this PR is trying to prevent again was that gems `rainbow` and `sass-rails` were removed which resulted in css not displaying on dev/staging.

In addition to the changes for `database.yml`, [config.assets.compile](https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/config/environments/production.rb#L32) has to be set to true otherwise you will receive an error message similar to what is below

```
Called from ~/.rbenv/versions/2.3.8/lib/ruby/gems/2.3.0/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:240:in `load_dependency'
F, [2019-08-27T13:50:23.154067 #60897] FATAL -- : [5e7d682f-e8c3-4426-bb08-a65c8894adea] [localhost]
ActionController::RoutingError (No route matches [GET] "/assets/application-acec3ea9bc666e8034fe06c893b1ed371b8185f4d95a8130ccaf86247d3ce043.js"):
```

Because changes are required to 2 files that are included in source control I went with this process instead as it does not affect any files in source control and tests what I was trying to test.

I titled the section "Local Production Dependency Build" to indicate this is specifically for running with production dependencies i.e. doing `bundle install --without development test;`